File: 0796-rotate-string/0796-rotate-string.cs
public class Solution {
    public bool RotateString(string s, string goal) {
        for(int i=s.Length-1;i>-1;i--)
        {
            if(goal.Equals(s.Substring(i,s.Length-i)+s.Substring(0,i)))
            {
                return true;
            }
        }
        return false;
    }
}

File: 0796-rotate-string/0796-rotate-string.java
class Solution {
    public boolean rotateString(String s, String goal) {
        
        if(s.length()!=goal.length())
        {
            return false;
        }

        for(int i=s.length()-1;i>-1;i--)
        {
            StringBuilder temp = new StringBuilder(s.substring(i,s.length()));
            if(goal.equals(temp.toString()+s.substring(0,i)))
            {
                return true;
            }
        }
        return false;
    }
}